app/views/club_applications/_club_application.text.erb

```diff
@@ -2,6 +2,7 @@ Name: <%= club_application.first_name %> <%= club_application.last_name %>
 High school: <%= club_application.high_school %>
 Year: <%= club_application.year %>
 Email: <%= club_application.email %>
+Phone Number: <%= if club_application.phone_number.blank? then 'N/A' else club_application.phone_number end %>
```

Comment:
No need for the "N/A" if it's blank. GitHub or Twitter can be blank and we don't set them to "N/A" if they're not set.